### DISCUSSION TITLE
Social Sharing App

### DISCUSSION THREAD

**Discussion by POSSENGINE** [GetPublii/Publii#1386] [Help]
Every time I turn on the social sharing app in the plugins section I get the following error when trying to preview or sync.

**Rendering process failed
An error occurred during rendering process: this.config.additionalSocialButtons.map is not a function**

Also appears to be no colouring options, is this done via the custom CSS section?

Pulling my hair out trying to work it out.

**Comment by bobmitro** [comment]
​This free support area is not available for commercial products; instead, please use a  dedicated Premium Support https://marketplace.getpublii.com/log-in/​
